src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -167,12 +172,14 @@ class CrossDeviceLinkUI extends Component {
     }
   }
 
-  mobileUrl = () =>
+  baseMobileUrl = () =>
     // This lets us test the cross device flow locally and on surge.
     // We use the same location to test the same bundle as the desktop flow.
     process.env.MOBILE_URL === "/" ?
-      `${window.location.origin}?link_id=${this.linkId}` :
-      `${process.env.MOBILE_URL}/${this.linkId}`
+      `${window.location.origin}?link_id=` :
+      `${process.env.MOBILE_URL}/`
+
+  mobileUrl = () => `${this.baseMobileUrl()}${this.linkId}`
```

Comment:
If I'm getting it right, we cannot do this. It opens a big hole, where someone could use our name to send people elsewhere. happy to chat about it.

Reply:
Agreed, I mentioned this to Stephen offline. Apart from the security issues, this will also hit the validations when trying to create the jwt. I don't think we should support this feature in our demo app, IMO the best way to go about this is for the acceptor to send a request via postman to create the jwt with the desired parameter.